tests/wpt/metadata/html/browsers/windows/auxiliary-browsing-contexts/opener-closed.html.ini

```diff
@@ -0,0 +1,3 @@
+[opener-closed.html]
+  [An auxiliary browsing context should report `null` for `window.opener` when that browsing context is discarded]
+    expected: TIMEOUT
```

Comment:
We're missing a TIMEOUT for the top-level test file, too:
```
[opener-closed.html]
  expected: TIMEOUT
  [An auxiliary browsing context should report `null` for `window.opener` when that browsing context is discarded]
    expected: TIMEOUT
```

Reply:
Fixed it with the latest push!